content/changelog/2025-01-25.md

```diff
@@ -0,0 +1,95 @@
+---
+description: A Neon Slack App, Neon Chat for VS Code, scheduled updates, and more
+---
+
+### The Neon Slack App is available for early access
+
+![Neon Slack App](/docs/relnotes/slack_app.png)
+
+The Neon Slack App helps you stay connected to your Neon Serverless Postgres databases. Here’s what you can do with it:
+
+- 📈 Track compute and storage usage in real time
+- 🔔 Receive alerts when your database approaches its performance limits
+- 🟢 Quickly verify the Neon platform's status
+
+We’d love to hear your feedback. Use the `/neon feedback` command in Slack to share your thoughts and feature requests.
+
+👉 [Install Neon Slack App](https://slack.com/apps/A083ZAHGL1Z) or check out the [documentation](/docs/manage/slack-app) to get started.
+
+### Neon Chat for Visual Studio Code
+
+The [Neon Chat for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=buildwithlayer.neon-integration-expert-15j6N) extension is now available in the GitHub Marketplace. This AI-powered assistant lets you chat with the latest Neon documentation without leaving your IDE.
+
+Get answers to common questions like:
+
+- _How to create a project?_
+- _How can I get started with the Neon API?_
+- _How do I create a branch using the Neon CLI?_
+
+![Neon Chat for Visual Studio Code](/docs/relnotes/neon_chat_visual_studio.png)
+
+### Scheduled updates coming soon 📅
+
+Neon is introducing scheduled updates, starting with Free Plan accounts and later expanding to Paid Plan accounts. These updates will cover Postgres updates, security patches, and Neon feature enhancements, automatically applied to your project's computes. Here's what to expect:
+
+- Updates aren’t new, but now they’ll be scheduled so you’ll know when they’re coming and won't fall behind on important maintenance.
+- Updates require a compute restart, but restarts are quick — taking just a few seconds.
+- You’ll be able to track scheduled updates in your project settings.
+- Free Plan accounts will have updates scheduled in advance for a specific day and time, while Paid Plan accounts will be able to choose a preferred update window.
+
+Stay tuned for specific details about when scheduled updates will roll out. Free Plan users can expect to see scheduled updates first, starting in early February. Scheduled updates on Paid Plans will roll out later, with updates for large compute sizes (> 8 CU) rolling out last.
+
+### Connect to external Postgres databases with `postgres_fdw` and `dblink`
```

Comment:
@myrrc can you please review this part?

Reply:
LGTM sans the "paid-only" part, asked in private

Reply:
As discussed, let's remove "paid-only" statement as these will be available on free plans